spec/board_spec.rb

```diff
@@ -0,0 +1,33 @@
+require './lib/board.rb'
+
+RSpec.describe 'testing the board' do
+  it 'should show us an empty board' do
+    expect(Board.new.show_board).to eq("+------------------------+\n| 1 - ❔  2 - ❔  3 - ❔ | \n| 4 - ❔  5 - ❔  6 - ❔ | \n| 7 - ❔  8 - ❔  9 - ❔ | \n+------------------------+")
+  end
+
+  it 'should show us an empty board' do
```

Comment:
This statement is the same as the previous test statement. Since this is a negative scenario test, the statement should reflect the actual test you are performing.

```ruby
it 'should not show a filled board' do
  # tests...
end
```

- [ ] Please update the test statements.